### PR TITLE
fixes prompting private rendered dicts

### LINKED ISSUE #1502: private rendered dict is still prompted
* Cookiecutter version: 2.0.0 from commit 52dd18513bbab7f0fbfcb2938c9644d9092247cf
* Python version: py3.8.7
* Operating System: Win

### Description:
For the `cookiecutter.json` below the private, rendered dict `__module_names` is still prompted in the output (last line).

For private variables c.f. https://github.com/cookiecutter/cookiecutter/issues/1115#issuecomment-442522474

### What I've run:

**cookiecutter.json:**
```
{
    "project_name": "Project Name",
    "project_slug": "{{ cookiecutter.project_name|lower|replace(' ', '-') }}",
    "__package_name": "{{ cookiecutter.project_name|lower|replace(' ', '_') }}",

    "__module_names": {
        "within": "{{ cookiecutter.project_slug }}",
        "outside": "{{ cookiecutter.__package_name }}"
    }
}
```
**verbose output:**
```
$ cookiecutter .\cookiecutter-context-test\ -f -v

DEBUG cookiecutter.config: User config not found. Loading default config.
DEBUG cookiecutter.main: context_file is .\cookiecutter-context-test\cookiecutter.json
DEBUG cookiecutter.generate: Context generated is OrderedDict([('cookiecutter', OrderedDict([('project_name', 'Project Name'), ('project_slug', "{{ cookiecutter.project_name|lower|replace(' ', '-') }}"), ('__package_name', "{{ cookiecutter.project_name|lower|replace(' ', '_') }}"), ('__module_names', OrderedDict([('within', '{{ cookiecutter.project_slug }}'), ('outside', '{{ cookiecutter.__package_name }}')]))]))])
project_name [Project Name]: 
project_slug [project-name]: 
__module_names [default]:
```

### PR DESCRIPTION
**Pull request by juhuebner**
fixes #1502

### CODE REVIEW

**Review by samj1912 — APPROVED** [review]

File: cookiecutter/utils.py [hunk 1]
```diff
@@ -12,9 +12,12 @@
 logger = logging.getLogger(__name__)
 
 
-def force_delete(func, path, exc_info):
+def force_delete(func, path, exc_info):  # pragma: no cover
```
Comment (simobasso) [inline cookiecutter/utils.py:15]: Please don't exclude this line. The CI have multiple os stage.

File: tox.ini [hunk 2]
```diff
@@ -37,7 +37,7 @@ passenv =
     HOME
 commands =
     pip install -e .
-    pytest --cov=cookiecutter --cov-report=term --cov-report=html --cov-report=xml
+    pytest --cov=cookiecutter --cov-report=term --cov-report=html --cov-report=xml {posargs}
```
Comment (simobasso) [inline tox.ini:40]: why this change?

File: cookiecutter/utils.py (hunk 1 above)
Comment (juhuebner) [reply cookiecutter/utils.py]: This has been a dev leftover. I removed it now.

File: tox.ini (hunk 2 above)
Comment (juhuebner) [reply tox.ini]: This has been a dev leftover. I removed it now.

**Review by simobasso — CHANGES_REQUESTED** [review]

File: cookiecutter/utils.py [hunk 3]
```diff
@@ -15,6 +15,9 @@
 def force_delete(func, path, exc_info):
     """Error handler for `shutil.rmtree()` equivalent to `rm -rf`.
 
+    Coverage is exluded since not all code is reached in all OSs due to
```
[the remark this answers is not in the document]
Comment (juhuebner) [reply cookiecutter/utils.py]: Oh, sorry. I was a bit to fast then. Sharp eyes 👓 Thanks!

**Review by simobasso — APPROVED** [review]

**Review by ssbarnea — APPROVED** [review]

**Comment by simobasso** [comment]
Thanks @juhuebner 🌮!
